logicrunner/builtin/builtin.go

```diff
@@ -106,13 +112,23 @@ func (b *BuiltIn) CallConstructor(
 	return constructorFunc(*objRef, args)
 }
 
-func (b *BuiltIn) CallMethod(ctx context.Context, callCtx *insolar.LogicCallContext, codeRef insolar.Reference,
-	data []byte, method string, args insolar.Arguments) ([]byte, insolar.Arguments, error) {
+func (b *BuiltIn) CallMethod(
+	ctx context.Context,
+	callCtx *insolar.LogicCallContext,
+	codeRef insolar.Reference,
+	data []byte,
+	method string,
+	args insolar.Arguments,
+) (
+	[]byte, insolar.Arguments, error,
+) {
 	executeStart := time.Now()
 	ctx = insmetrics.InsertTag(ctx, metrics.TagContractPrototype, b.PrototypeRefRegistry[codeRef])
-	ctx = insmetrics.InsertTag(ctx, metrics.TagContractMethodName, method)
+	ctx = insmetrics.InsertTag(ctx, metrics.TagContractMethodName, "Constructor")
```

Comment:
You may have multiple constructors with different names. I don't mind a prefix or something else added here.  Please split out these changes to metrics from this PR. They have nothing to do with recovering panics in contracts.

Reply:
fixed